src/ploneorg/addonlisting/browser/views.py

```diff
@@ -38,3 +39,17 @@ class AddOnView(BrowserView):
 
     def __call__(self):
         return self.template()
+
+class AddOnFolderView(BrowserView):
+    """ A list of addons
+    """
+
+    def addons(self):
+        results = []
+        brains = api.content.find(context=self.context, portal_type='AddOn')
+        for brain in brains:
```

Comment:
if you just want to have those existing fields title and description you could directly return the brains.